src/components/medication-event.component.tsx

```diff
@@ -37,44 +37,45 @@ const MedicationEvent: React.FC<{
           )}
         </strong>
       </p>
-      <p className={styles.bodyLong01}>
-        <span className={styles.label01}>
-          {t("dose", "Dose").toUpperCase()}
-        </span>{" "}
-        {dosageInstruction && (
-          <>
-            <span className={styles.dosage}>
-              {dosageInstruction.doseAndRate &&
-                dosageInstruction?.doseAndRate.map((doseAndRate, index) => {
-                  return (
-                    <span key={index}>
-                      {doseAndRate?.doseQuantity?.value}{" "}
-                      {doseAndRate?.doseQuantity?.unit}
-                    </span>
-                  );
-                })}
-            </span>{" "}
-            &mdash; {dosageInstruction?.route?.text} &mdash;{" "}
-            {dosageInstruction?.timing?.code?.text}{" "}
-            {dosageInstruction?.timing?.repeat?.duration
-              ? "for " +
-                dosageInstruction?.timing?.repeat?.duration +
-                " " +
-                dosageInstruction?.timing?.repeat?.durationUnit
-              : " "}
-          </>
-        )}
-      </p>
-      <p className={styles.bodyLong01}>
-        <span className={styles.label01}>
-          {t("quantity", "Quantity").toUpperCase()}
-        </span>{" "}
-        {quantity && (
+
```

Comment:
just some changes so that labels aren't displayed in the medication event card if there isn't any associated content